src/main.js

```diff
@@ -92,18 +96,60 @@ function main() {
 			errorCounter = 0;
 		}
 		log("Waiting for own turn, sleep 1 second.");
-		currentActionOutput.value = "Waiting for own turn.";
+		clearCrtStrategyMsg();
+		showCrtActionMsg("Waiting for own turn.");
 		setTimeout(main, 1000);
+
+		if (MODE === AIMODE.HELP) {
+			oldOps = [];
+		}
+
 		return;
 	}
 
-	currentActionOutput.value = "Calculating best move...";
```

Comment:
I think moving this from here into the mainOwnTurn function is not going to work. The browser won't refresh the GUI element while the javascript thread is blocking it. Only if this is called before the setTimeout() the browser has time to update it.

Reply:
I revert this code back and it looks better now.